dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -281,6 +287,63 @@ private static boolean problemFilter(Map.Entry<SymbolProblem, ClassFile> entry)
     }
   }
 
+  /**
+   * Returns an {@code Optional} describing the linkage error for the interface if any methods of
+   * the interface are not implemented in the referencing {@code classFile}. Such unimplemented
+   * methods manifest as {@link AbstractMethodError} in runtime.
+   */
+  private Optional<SymbolProblem> findInterfaceProblem(
```

Comment:
Why is this an optional instead of a list? Can there only be one such problem? What if there's more than one missing method?

Reply:
It's not a list because it reports one error for one or more unimplemented methods for the interface.

Reply:
This doesn't feel write. Two missing methods should be two errors. Can this return a list instead?

Reply:
Now it reports all missing method symbols on the class.